projects/ad405x_iio/scripts/ad405x_data_capture.py

```diff
@@ -0,0 +1,160 @@
+import numpy
+from serial import Serial
+from time import sleep
+from pynput import keyboard
+import sys
+import select
+import os
+import csv
+import math
+from ad405x import *
+
+# Global variables
+line = 0
+writer = 0
+run_continuous = False
+iterations = 0
+data_capture_abort = False
+chn_count = 0
+data_list = []
+
+def key_press_event(key):
+    global data_capture_abort
+    data_capture_abort = True
+
+def init_data_capture():
+    global device
+    global data_list
+    global chn_count
+    global listener
+    global samples_block
+
+    ######## User configuration ##########
+    # Configure the backend for PC to IIOD interface
+    uri = "serial:COM16,230400"  # For UART, baud rate must be same as set in the FW. COM port is physical Or VCOM.
+    device_name = "ad4052"      # Name of the device must be same as set in the FW.
+    ######################################
+
+    # Create an IIO device context
+    device = ad405x(uri, device_name)
+    device._ctx.set_timeout(100000)
+
+    ######## User configuration ##########
+    # Channels to be captured e.g. ["voltage0"]: 1chn, ["voltage0, voltage1"]: 2chns
```

Comment:
Isn't this a single channel part?
Is this comment applicable here?

Reply:
Yeah, it's indeed a single channel part. Will amend the comment

Reply:
Modified it for the AD4052 case